store/sqlstore/user_store.go

```diff
@@ -325,6 +325,24 @@ func (us SqlUserStore) UpdateMfaActive(userId string, active bool) error {
 	return nil
 }
 
+func (us SqlUserStore) GetMany(ids []string) ([]*model.User, error) {
```

Comment:
Please add a godoc to this, and some store tests. :)

Reply:
Amended in https://github.com/mattermost/mattermost-server/pull/16819/commits/3ba1726037e26841ecf2b9ca8afd5e6d8a289c7a